test/e2e/case38_install_operator_test.go

```diff
@@ -550,4 +550,110 @@ var _ = Describe("Test installing an operator from OperatorPolicy", Ordered, fun
 			)
 		})
 	})
+	Describe("Test status reporting for catalogsource", Ordered, func() {
+		const (
+			OpPlcYAML = "../resources/case38_operator_install/operator-policy-with-group.yaml"
+			OpPlcName = "oppol-with-group"
+			subName   = "project-quay"
+		)
+
+		BeforeAll(func() {
+			By("Applying creating a ns and the test policy")
+			utils.Kubectl("create", "ns", opPolTestNS)
+			DeferCleanup(func() {
+				utils.Kubectl("delete", "ns", opPolTestNS)
+			})
+
+			createObjWithParent(parentPolicyYAML, parentPolicyName,
+				OpPlcYAML, opPolTestNS, gvrPolicy, gvrOperatorPolicy)
+		})
+
+		It("Should initially be Compliant", func() {
```

Comment:
Nit:
```suggestion
		It("Should initially show the CatalogSource is compliant", func() {
```
just to be more accurate, since the policy overall is not compliant.